### PR TITLE
Update README.md with information about preprint

### PR DESCRIPTION
**Pull request by davidbbeyer**
Addresses #49 

@pm-blanco I have added a short note about the preprint and how to cite it to the Readme. My suggestion would be to merge this, but leave the issue open as a reminder (e.g. to replace the preprint by the paper eventually).

### CODE REVIEW

**Review by pm-blanco — CHANGES_REQUESTED** [review]
I agree with merging this PR now. I would leave the issue open not only as a reminder to update this info once we publish the article, but also to address the cite feature proposed in #5

File: README.md [hunk 1]
```diff
@@ -159,3 +159,21 @@ folder and a corresponding line in the `testsuite` target of the Makefile.
 
 Every contribution is automatically tested in CI using EESSI (https://www.eessi.io)
 and the [EESSI GitHub Action](https://github.com/marketplace/actions/eessi).
+
+## References
+
+Check out the corresponding [preprint](https://doi.org/10.48550/arXiv.2401.14954) to learn more about pyMBE.
+If you use pyMBE in your research, make sure to cite the preprint:
```
[the remark this answers is not in the document]
Comment (davidbbeyer) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -159,3 +159,21 @@ folder and a corresponding line in the `testsuite` target of the Makefile.
...
+@article{beyer2024pymbe,
+  title={pyMBE: the Python-based Molecule Builder for ESPResSo},
+  author={Beyer, David and Torres, Paola B and Pineda, Sebastian P and Narambuena, Claudio F and Grad, Jean-No{\"e}l and Ko{\v{s}}ovan, Peter and Blanco, Pablo M},
+  journal={arXiv preprint arXiv:2401.14954},
+  year={2024},
+  doi={10.48550/arXiv.2401.14954}
+}
+```
+
+Also make sure to properly cite the corresponding papers if you use the $\mathrm{p}K_{\mathrm{A}}$-sets or interaction parameters provided in the `parameters/` folder.
```
[the remark this answers is not in the document]
Comment (davidbbeyer) [reply README.md]: done

**Review by pm-blanco — APPROVED** [review]
Looks good to me

**Pull request merged**